sw/airborne/modules/imu/imu.c

```diff
@@ -164,6 +182,8 @@ static void send_accel_raw(struct transport_tx *trans, struct link_device *dev)
   static uint8_t id = 0;
   pprz_msg_send_IMU_ACCEL_RAW(trans, dev, AC_ID, &imu.accels[id].abi_id, &imu.accels[id].temperature,
                               &imu.accels[id].unscaled.x, &imu.accels[id].unscaled.y, &imu.accels[id].unscaled.z);
+  if(imu.accel_abi_send_id != ABI_DISABLE && imu.accel_abi_send_id != ABI_BROADCAST && imu.accels[id].abi_id == imu.accel_abi_send_id)
```

Comment:
Here it seems that DISABLE and BROADCAST have the same effect (send data in a rolling way). What about using the DISABLE to actually disable the sending of messages ? Does it makes sense ?

Reply:
Yes I thought about that as well will implement it like that